### PR TITLE
Add path field to imprint response

### PR DESCRIPTION
**Pull request by ulliholtgrave**
### Short description

This PR is correcting the path and url field in the api response for the imprint

Fixes: #1227

### CODE REVIEW

**Review by svenseeberg — APPROVED** [review]
Works. Changelog still missing.

File: integreat_cms/api/v3/imprint.py [hunk 1]
```diff
@@ -18,7 +19,8 @@ def transform_imprint(imprint_translation):
     """
     return {
         "id": imprint_translation.id,
-        "url": imprint_translation.get_absolute_url(),
+        "url": settings.BASE_URL + imprint_translation.get_absolute_url(),
+        "path": imprint_translation.get_absolute_url(),
```
Comment (timoludwig) [inline integreat_cms/api/v3/imprint.py:23]: I believe `get_absolute_url()` needs a database request, so probably it should be executed only once and be stored to a variable?

Comment (ulliholtgrave) [reply integreat_cms/api/v3/imprint.py:23]: Okay... :joy:

**Review by timoludwig — APPROVED** [review]
:+1:

**Pull request merged**